app/utils/pusher.js

```diff
@@ -55,7 +55,7 @@ TravisPusher.prototype.init = function (config, ajaxService) {
   });
 
   if (config.path) {
-    this.pusher.wsPath = `/${config.path}`,
+    this.pusher.wsPath = `/${config.path}`;
```

Comment:
![image](https://user-images.githubusercontent.com/47749/37531856-c5be2df0-2913-11e8-9602-f5a072dfa952.png)

Beat me to it again. :D

Reply:
haha it’s because I moved the original line down and didn’t notice that 😆